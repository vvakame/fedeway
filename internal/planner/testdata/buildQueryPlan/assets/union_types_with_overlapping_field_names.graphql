# schema: supergraphSdl.graphqls

# should not confuse union types with overlapping field names
query {
    body {
        ... on Image {
            attributes {
                url
            }
        }
        ... on Text {
            attributes {
                bold
                text
            }
        }
    }
}
