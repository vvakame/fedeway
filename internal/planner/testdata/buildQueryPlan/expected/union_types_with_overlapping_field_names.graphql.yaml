node:
  servicename: documents
  variableusages: []
  requires: []
  operation: |
    query {
    	body {
    		__typename
    		... on Image {
    			attributes {
    				url
    			}
    		}
    		... on Text {
    			attributes {
    				bold
    				text
    			}
    		}
    	}
    }
